src/schemas/auth.js

```diff
@@ -3,12 +3,14 @@ const { gql } = require('apollo-server-express');
 const authTypeDefs = gql`
   extend type Query {
     getActiveUser: User
+    sendPasswordEmail(email:String!): Boolean
```

Comment:
`sendResetPasswordEmail` maybe? though this would require F/E change too, nbd if you don't update this